starter-code/pseudocode.txt

```diff
@@ -52,8 +52,8 @@ func arrayMax(var A as array of length n)
     return currentMax
 end func
 
-Running time: 
-Explanation: 
+Running time: O(1)
+Explanation: The operation just looks for a single index and returns the value of the index directly; so the operation time will be constant all the time.
```

Comment:
The work done _inside_ each loop iteration is O(1), but there are still n iterations, so overall it'll be O(n)